packages/nc-gui/components/cell/DateTimePicker.vue

```diff
@@ -25,8 +26,11 @@ interface Props {
 }
 
 const { modelValue, isPk, isUpdatedFromCopyNPaste } = defineProps<Props>()
-
 const emit = defineEmits(['update:modelValue'])
+dayjs.extend(updateLocale)
+dayjs.updateLocale('en', {
+  weekStart: 1,
+})
```

Comment:
I would prefer to move it to `packages/nc-gui/plugins/a.dayjs.ts` rather than repeating.